index.js

```diff
@@ -20,25 +18,46 @@ function get_new_question() {
         let quote_element = document.getElementById('quote')
         quote_element.textContent = current_movie.quote
         let answer_element = document.getElementById('answer')
+        answer_element.dataset.year = current_movie.year
+        answer_element.dataset.title = current_movie.title
         answer_element.style.display = 'none'
         answer_element.innerHTML = '<h2>' + current_movie.title + '</h2>'
         let correct_incorrect_elem = document.getElementById('correct_incorrect')
         correct_incorrect_elem.style.display = 'none'
     }
 }
 
+function get_hint () {
+    fetch('src/films.json')
+        .then(data => data.json())
+        .then((data) => {
+            let answer_year = document.getElementById('answer').dataset.year
+            let answer_title = document.getElementById('answer').dataset.title
+            let movie_titles = (data.films).map(movie => movie.title)
+            let filtered_movies =  movie_titles.filter(movie => movie !== answer_title)
+            let hint = shuffle_array(filtered_movies).slice(0, 2)
+            hint.push(answer_title)
+            shuffle_array(hint)
+            hint.push(answer_year)
+            document.getElementById('hint').innerHTML = '<p>This Movie was released in: ' + hint[3] + '</p> <ul><li>' + hint[0] + '</li><li>' + hint[1] + '</li><li>' + hint[2] + '</li></ul>'
```

Comment:
I don't love how this array is storing different things, it relies on you knowing that the key of 3 contains the year. Would be better to use an object with keys instead to make this easier to understand/work on in future